shells/backends/gcp_tf_backend/src/driver.py

```diff
@@ -60,38 +59,39 @@ def get_inventory(self, context):
         resource.add_sub_resource('1', port1)
         return resource.create_autoload_details()
         '''
-        gcp_service = self._can_connect(context)
-        if not gcp_service:
-            raise ValueError("Can't connect to GCP")
+        logger = LoggingSessionContext(context)
```

Comment:
this will not work as expected. Need to use it with the "with" statement.

Reply:
thank you, will add it